setup.py

```diff
@@ -364,6 +370,12 @@ def run(self):
     # https://github.com/pandas-dev/pandas/issues/35559
     extra_compile_args.append("-Wno-error=unreachable-code")
 
+    # rdata requires system iconv library
+    os.environ["DYLD_LIBRARY_PATH"] = ""
```

Comment:
Overriding the users DYLC_LIBRARY_PATH feels like the wrong solution.  Why is this needed? Does this change this path for the duration of the console session, even outside of the setup run?

Reply:
This line is not needed and has been removed.